src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,113 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+
+    private Entry<K, V>[] table;
+    private int size = 0;
```

Comment:
0 is default int value, no need to explicitly set it
```suggestion
    private int size;
```